engine/menus/save.asm

```diff
@@ -37,6 +37,7 @@ LoadSAV0:
 	ld a, $1
 	ld [MBC1SRamBankingMode], a
 	ld [MBC1SRamBank], a
+	vc_hook SaveLimit ; This hook is ambigiously located here. Can be Moved.
```

Comment:
```suggestion
; This vc_hook does not have to be in any particular location.
; It is defined here because it refers to the same labels as the two lines below.
	vc_hook SaveLimit
```